core/unit_test/incremental/Test12a_ThreadScratch.hpp

```diff
@@ -93,7 +99,7 @@ struct ThreadScratch {
     int scratchSize = scratch_t::shmem_size(sY);
     // So this works with deprecated code enabled:
     policy_t policy =
-        policy_t(pN, Kokkos::AUTO)
+        policy_t(pN, Kokkos::AUTO, 1)
```

Comment:
Did you mean that change?  Why is this ok?

Reply:
It doesn't change anything. The default vector length is 1 (see https://github.com/kokkos/kokkos/blob/9cf0d5bc02513aece0a79c12d717714cf9345700/core/src/Cuda/Kokkos_Cuda_Parallel.hpp#L288-L291, https://github.com/kokkos/kokkos/blob/9cf0d5bc02513aece0a79c12d717714cf9345700/core/src/HIP/Kokkos_HIP_Parallel_Team.hpp#L239-L242, and https://github.com/kokkos/kokkos/blob/9cf0d5bc02513aece0a79c12d717714cf9345700/core/src/SYCL/Kokkos_SYCL_Parallel_Team.hpp#L203-L206) and I wanted to stress that to justify replacing `ThreadVectorRange` with a serial loop.